pkg/cloud/volume_limits.go

```diff
@@ -131,215 +131,337 @@ func GetNVMeInstanceStoreVolumesForInstanceType(it string) int {
 	return 0
 }
 
-// / https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-volumes
+// / https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-store-volumes.html
 // / IMDS does not provide NVMe instance store data; we'll just list all instances here
 // / TODO: See if we can get these values from DescribeInstanceTypes API
+// Generated by command, requires aws cli and jq:
+// aws ec2 describe-instance-types --region us-east-1 --filters "Name=instance-storage-supported,Values=true" --query "InstanceTypes[].[InstanceType, InstanceStorageInfo]" | jq -r 'map("\"" + .[0] + "\": " + (.[1].Disks | map(.Count) | add | tostring) + ",") | .[]' | sort
```

Comment:
This is great, thank you! Should we consider adding to our release runbook?

Reply:
Yes, but not relevant to this PR, resolving